src/robusta/model/playbook_definition.py

```diff
@@ -1,44 +1,40 @@
 import hashlib
+import math
+import json
+from os import name
 from typing import Any, Dict, List
-
-from pydantic import BaseModel, PrivateAttr
+from pydantic import BaseModel, PrivateAttr, validator
 
 from robusta.core.playbooks.trigger import Trigger
 from robusta.model.playbook_action import PlaybookAction
 
 
 class PlaybookDefinition(BaseModel):
-    sinks: List[str] = None
+    name: str
+    sinks: List[str] = None  # compatibility
     triggers: List[Trigger]
-    actions: List[Dict[str, Any]]
-    _actions: List[PlaybookAction] = PrivateAttr()
-    stop: bool = False
-    _playbook_id: str = PrivateAttr()
+    actions: List[PlaybookAction] = []
+    tags: Dict[str, str] = {}
```

Comment:
I think we don't use tags anywhere.
If that's the case, maybe it's better to remove it